x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/threshold/threshold.ts

```diff
@@ -152,7 +151,10 @@ export const thresholdExecutor = async ({
       ruleExecutionLogger,
     });
 
-    addToSearchAfterReturn({ current: result, next: createResult });
+    addToSearchAfterReturn({
+      current: result,
+      next: { ...createResult, success: createResult.success && isEmpty(searchErrors) },
```

Comment:
After my changes to the find_threshold_signals file I was surprised that the status never changed to failing. Turns out that the success status only depended on the success of the bulk creation of signals and disregarded search errors.